lib/chef/resource/rhsm_register.rb

```diff
@@ -60,6 +60,10 @@ class RhsmRegister < Chef::Resource
         description: "If true, the system will be registered even if it is already registered. Normally, any register operations will fail if the machine has already been registered.",
         default: false, desired_state: false
 
+      property :use_https, [TrueClass, FalseClass],
+        description: "If true, chef will fetch the katello-ca-consumer-latest.noarch.rpm from the satellite_host using HTTPS.",
+        default: false, desired_state: false
```

Comment:
should probably flip the default to true.  this is also a very generic sounding name of a property for a very specific function.  it should probably be `use_katello_https` or something like that.

Reply:
also maybe consider just fixing this to be hardcoded?  would anyone really not want this?

Reply:
Out of the box internal Satellite servers use a self-signed certificate. Red Hat themselves even reference http (as opposed to https) urls for installation of katello-ca-consumer RPM.

Reference: https://access.redhat.com/documentation/en-us/red_hat_satellite/6.6/html-single/quick_start_guide/index#registering_existing_system

Reply:
> should probably flip the default to true. this is also a very generic sounding name of a property for a very specific function. it should probably be `use_katello_https` or something like that.

Agreed. I can't think of a short name that gives a clear gist of what it is for that isn't huge. Also, it's not katello, it's the katello-ca-consumer package which are different things.

Reply:
`https_for_ca_consumer true` ?

Reply:
> `https_for_ca_consumer true` ?

works for me. I'll update and re-push in a bit.